src/components/Drawer/stories.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from 'react';
+import { button, select } from '@storybook/addon-knobs';
+import Drawer from './index';
+
+
+export default {
+    Component: Drawer,
+    title: 'Components|Display/Drawer',
+};
+
+
+export const controlled = (): ReactElement => {
```

Comment:
Since there is only a controlled version, we should rename this `Default`

Reply:
fixed